techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -108,51 +109,94 @@ struct SynthQuickLogicPass : public ScriptPass
         }
 
         if (check_label("coarse")) {
+            run("opt_expr");
+            run("opt_clean");
+            run("check");
+            run("opt");
             run("wreduce -keepdc");
             run("peepopt");
             run("pmuxtree");
-            run("opt");
-            run("memory -nomap");
             run("opt_clean");
+            run("share");
+            run("techmap -map +/cmp2lut.v -D LUT_WIDTH=4");
+            run("opt_expr");
+            run("opt_clean");
```

Comment:
not sure if we really need so many opts here

Reply:
I can test with and without this, if there is any difference in the utilisation. If no difference, I will remove them.

Reply:
there is no difference so removing these optimisations.